packages/next/src/root-layout/index.tsx

```diff
@@ -30,3 +32,26 @@ export async function RootLayout( {
 		</html>
 	);
 }
+
+/**
+ * Generate and return root metadata, including icons and other metadata.
+ *
+ * @return Merged metadata.
+ */
+export async function generateRootMetaData(): Promise< Metadata > {
+	// Fetch icons in required format, apply faviconIcons and apple touch icons in icons metadata property while apply msapplication-TileImage in other metadata property
```

Comment:
Documenting https://github.com/rtCamp/snapwp/pull/119#issuecomment-2736076355

```suggestion
	/**
	 * Fetch icons in required format, apply faviconIcons and apple touch icons in icons metadata property while apply msapplication-TileImage in other metadata property.
	 *
	 * @todo Review composability when implementing SEO metadata
```